.github/workflows/scrape-fj-manual.yml

```diff
@@ -19,8 +19,11 @@ jobs:
 
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
-          - name: checkout repo content
-            uses: actions/checkout@v2 # checkout the repository content to github runner.
+          - name: sparse checkout country file directory
+            uses: snow-actions/sparse-checkout@v1.1.0
+            with:
+              patterns: ./mcd-sel-fj
```

Comment:
Checkout only the required folder for scraping rather than the whole repository (files at root are also pulled).